test/known_issues/hooks-thread/test-hooks-thread-deadlock.js

```diff
@@ -0,0 +1,42 @@
+const { mustNotCall } = require('../../common/index.js');
+const assert = require('node:assert');
+const { spawn } = require('node:child_process');
+const path = require('node:path');
+const { execPath } = require('node:process');
+const { describe, it } = require('node:test');
+
+describe('hooks deadlock', { concurrency: true }, () => {
+  it('will deadlock when a/hooks…resolve tries to contact the main thread during b/register', async () => {
+    let stderr = '';
+    let stdout = '';
+    // ! Do NOT use spawnSync here: it will deadlock.
+    const child = spawn(execPath, [
```

Comment:
```suggestion
    const child = await spawnPromisified(execPath, [
```

Reply:
How do I then verify it is indeed "broken"? `spawnPromisified` will cause the process to hang.

Reply:
I would call that broken, isn’t that good enough?

Reply:
No, I mean it would break CI. The test as currently written demonstrates that the targeted behaviour is broken. Do these tests work differently?

Reply:
Yeah, `known_issues` are for tests that are not passing (but we'd like them to). E.g. `out/Release/node test/known_issues/test-vm-ownkeys.js` exits with non-zero code, and `tools/test.py test/known_issues/test-vm-ownkeys.js` shows "All tests passed".